src/OmniSharp.Abstractions/OmniSharpEndpoints.cs

```diff
@@ -60,6 +60,8 @@ public static class V2
 
             public const string BlockStructure = "/v2/blockstructure";
             public const string CodeStructure = "/v2/codestructure";
+
+            public const string SemanticHighlight = "/v2/semantichighlight";
```

Comment:
I think we can just call it `/v2/highlight`, as in an improved version of the existing classification
it's safe to assume that there won't be a v2 of the existing highlighting endpoint - we will only have this one

Reply:
Sure thing